src/Illuminate/Collections/Arr.php

```diff
@@ -744,4 +744,30 @@ public static function wrap($value)
 
         return is_array($value) ? $value : [$value];
     }
+
+    /**
+     * Returns the item value converted to integer.
+     *
+     * @param  \ArrayAccess|array  $array
+     * @param  string|int|null  $key
+     * @param  mixed  $default
+     * @return int
+     */
+    public static function getInt($array, $key, $default = null): int
```

Comment:
I'd expect `$default` to be returned when the array key doesn't exist, yet it will return `0` instead of `null`

Reply:
` $default = null` should IMO be ` int $default = 0` as the whole point of this method is to always return an integer so allowing the `mixed` type for `$default` does not make sense IMO.